filebeat/harvester/log.go

```diff
@@ -240,15 +236,9 @@ func (h *Harvester) getState() file.State {
 		return file.State{}
 	}
 
-	h.refreshState()
-	return h.state
-}
-
-// refreshState refreshes the values in State with the values from the harvester itself
-func (h *Harvester) refreshState() {
-	h.state.Source = h.path
-	h.state.Offset = h.getOffset()
+	// refreshes the values in State with the values from the harvester itself
 	h.state.FileStateOS = file.GetOSState(h.state.Fileinfo)
+	return h.state
```

Comment:
About harvester stdin/log: Yes, it should be 2 different harvesters in the future.